tutorials/intermediate/artists.py

```diff
@@ -97,11 +95,11 @@ class in the Matplotlib API, and the one you will be working with most
 
 If you make subsequent calls to ``ax.plot`` (and the hold state is "on"
 which is the default) then additional lines will be added to the list.
-You can remove lines later simply by calling the list methods; either
-of these will work::
+You can remove a line later by calling its ``remove`` method::
 
-    del ax.lines[0]
-    ax.lines.remove(line)  # one or the other, not both!
+    # One or the other, not both!
+    ax.lines[0].remove()
+    line.remove()
```

Comment:
```suggestion
    line = ax.lines[0]
    line.remove()
```
Prevents the need for "one or the other". If people already have a Line object they can trivially see that the don't need the first line.